react-app/public/index.html

```diff
@@ -10,6 +10,7 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
```

Comment:
```suggestion
```

We should not need to use CDNs for CSS or JS content in `index.html`.

Since we set up the react app with `create-react-app`, we should be able to install these dependencies with just `npm install --save ___`

For bootstrap CSS specifically in this line, you can follow what the [React bootstrap docs](https://react-bootstrap.github.io/getting-started/introduction#installation) say for using `npm` to install the library